.github/images/rit-local-configs/regtest-all-keys.js

```diff
@@ -0,0 +1,116 @@
+const path = require('path');
+const nodesConfigPath = 'config/node-configs';
+const keysPathResolve = 'node-keys';
+const classpath = process.env.POWPEG_NODE_JAR_PATH;
+const federatesLogbackPath = path.resolve(__dirname, 'logback');
```

Comment:
```suggestion
const federatesLogbackPath = path.resolve(__dirname, 'logbacks');
```

Reply:
Or, change the `logbacks` part of the `.github/images/rit-local-configs/logbacks/logback-fed-5.xml` path to look like `.github/images/rit-local-configs/logback/logback-fed-5.xml`. It looks like it has an extra 's' in it.

Reply:
Applied your suggestion. Thanks.